### ISSUE TITLE
monaca command not working

### ISSUE THREAD

**Issue by jdtzmn** [monaca/monaca-cli#25]
This is what happens when I try running monaca:

```
$ sudo npm install -g monaca
$ monaca
: No such file or directory

```

That's all it says. I've also tried `$ sudo monaca` but to no avail...

**Comment by masahirotanaka** [comment]
It's fixed now and released as 1.2.1.
Thanks so much for the feedback!

**Comment by jdtzmn** [comment]
The issue is still not fixed... Just updated today
